Sources/StreamChatUI/Utils/ImageLoading/NukeImageLoader.swift

```diff
@@ -0,0 +1,42 @@
+//
+// Copyright © 2021 Stream.io Inc. All rights reserved.
+//
+
+import Nuke
+import UIKit
+
+extension ImageTask: Cancellable {}
```

Comment:
this is internal by default, please check do we need to do this `public extension`?

Reply:
We do not need this to be public